### DISCUSSION TITLE
重新检测余额，这个按钮不能点击

### DISCUSSION THREAD

**Discussion by Dashan-IZ** [ChatGPTNextWeb/NextChat#1033] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/78125220/233839457-468866a2-b1dd-470e-b7af-998e0bf47024.png)
bug，不知道是设计还是bug，不能重复点击

**Comment by ClarenceDan** [comment, marked as answer]
不是BUG，上次更新后设置了查询后一分钟的冷却时间，如果不喜欢可以修改 update.ts里的时间限制
